include/boost/geometry/strategies/convex_hull/cartesian.hpp

```diff
@@ -0,0 +1,49 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CONVEX_HULL_CARTESIAN_HPP
+
+
+#include <boost/geometry/strategies/convex_hull/services.hpp>
+#include <boost/geometry/strategies/detail.hpp>
+#include <boost/geometry/strategy/cartesian/side_robust.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace convex_hull
+{
+
+template <typename CalculationType = void>
+class cartesian : strategies::detail::cartesian_base
```

Comment:
I forgot to mention that the base class should be `public` in order to allow access to its members.

Reply:
It is `public`. Am I missing something or you are looking at a older version?

Reply:
No, I don't think so. It should either be:
```
class cartesian : public strategies::detail::cartesian_base
```
or
```
struct cartesian : strategies::detail::cartesian_base
```

Reply:
oh, sure, I misunderstood your first comment